lib/src/components/NxGlobalSidebar/types.ts

```diff
@@ -53,3 +53,31 @@ export const nxGlobalSidebarNavigationLinkPropTypes: ValidationMap<NxGlobalSideb
   text: PropTypes.string.isRequired,
   href: PropTypes.string.isRequired
 };
+
+export interface NxGlobalSidebarFooterProps {
+  showSupport: boolean;
+  supportIcon?: IconDefinition;
```

Comment:
No `| null` on this one?

Reply:
> No `| null` on this one?

Now there is: https://github.com/sonatype/sonatype-react-shared-components/pull/387/commits/0e6dd7b2dbf27a77c724a8c498aee6a862550936